modules/ROOT/pages/to-find-info.adoc

```diff
@@ -30,54 +13,131 @@ Exchange Search looks for terms in:
 * The content of the asset portal pages
 * xref:to-find-info.adoc#searching-api-specifications[Properties defined in RAML and OAS API specifications]
 
-You can combine search terms with the filters listed in All Types.
+In Exchange you can enter search text to search for assets, click *All Types* to set a filter for specific asset types, or combine search text with the filters listed in the type menu.
+
+<<index.adoc#content-types,Content Types>> lists the different types of asset.
+
+You can also search by clicking category or tag elements. This creates a filtered search you can use to find similar assets.
+
+The original Exchange search experience is avilable in Anypoint Platform Private Cloud Edition, MuleSoft Government Cloud, the US cloud, and the EU cloud. The new Exchange search experience is available in the US cloud and the EU cloud.
 
-You can also search by clicking Category or Tag elements. This creates a filtered search you can use to find similar assets.
+In the original Exchange search experience, searches apply only to the Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
 
-In Exchange Private Cloud Edition and GovCloud, searches apply only to the Anypoint Exchange organization you are viewing. If you search in a private Exchange organization, the search only lists that organization's assets and not MuleSoft's public assets.
+In the new Exchange search experience, you can select which of the organizations visible to you should be included in each search.
 
-In Exchange Cloud, you can select which of the organizations visible to you should be included in each search.
+To search by tags and categories in the new Exchange search experience, use the drop-down menus for tags and categories on the search page.
```

Comment:
You can also add a section about sorting feature on the right side of the search panel. Something like - results are sorted by "Relevance" by default. The matching results are ranked for relevance based on where a match is found in the asset metadata such as name, title, pages and endpoints.

Reply:
Good point! Added a description of the sorting menu based on your text here.